openapi/components/schemas/StorefrontQuoteReactivateOrder.yaml

```
@@ -0,0 +1,481 @@
+type: object
+required:
+  - websiteId
+  - customerId
+  - items
+properties:
+  id:
+    readOnly: true
+    description: ID of the quote.
+    type: string
+    maxLength: 50
+    example: qt_0YV7DES3WPC5J8JD8QTVNZBZNZ
+  type:
+    description: Specifies the type of the quote. A quote can be a subscription or a one-time purchase.
+    type: string
+    readOnly: true
+    enum:
+      - subscription-order
+      - one-time-order
+  action:
+    description: |-
+      Action of the quote for churned/canceled order reactivation.
+      On accept quote new invoice is issued to be and linked to `invoiceId`. Quote items are applied to the order.
+      The related order will have `pending` status.
```

suggestion: for readability 
```suggestion
      Action of the quote for churned or canceled order reactivation.
      When the accept quote action is used,
      a new invoice is issued to be and linked to the `invoiceId`. Quote items are applied to the order.
      The related order will have the `pending` status.
```